Sources/ClientRuntime/Networking/Http/URLSession/URLSessionHTTPClient.swift

```diff
@@ -502,34 +502,37 @@ public final class URLSessionHTTPClient: HTTPClient {
                     body = .data(nil)
                 }
 
+                // Capture current time, then spawn a Task to write HTTP telemetry
```

Comment:
Code below is the same as before, except:
- Accesses to `telemetry` and `telemetry. httpMetricsUsage` are now async since those properties are now actors.
- Telemetry code is moved into a spawned Task, which allows for async operations and allows telemetry to be recorded concurrently with continuing the request.